src/components/VProgressLinear/VProgressLinear.js

```diff
@@ -115,12 +137,19 @@ export default {
     const fade = h('v-fade-transition', [this.indeterminate && this.genIndeterminate(h)])
     const slide = h('v-slide-x-transition', [!this.indeterminate && this.genDeterminate(h)])
 
-    const bar = h('div', { class: ['progress-linear__bar', this.colorBack], style: this.styles }, [fade, slide])
+    const bar = h('div', { class: ['progress-linear__bar'], style: this.styles }, [fade, slide])
```

Comment:
this class should change to **staticClass**. (I know it was like this before, but you're in here :))